oauth2client/client.py

```diff
@@ -844,6 +834,19 @@ def _generate_refresh_request_headers(self):
 
         return headers
 
+    def _initialize_headers(self, headers):
```

Comment:
Does this behavior have to be an instance-scope method that plays a role in the inheritance chain? It barely uses its `self` parameter. Can it be a library-private helper function?